boards/olimex-msp430-h2618/doc.txt

```diff
@@ -93,19 +93,19 @@ make BOARD=olimex-msp430-h2618 flash
 
 ## Using the shell
 
-stdio is available via the UART interface with `TXD = P3.4`
-(pin 35 on the header) and `RXD = P3.5` (pin 34 on the header) at 115,200 Baud.
+stdio is available via the UART interface with `TXD = P3.6`
+(pin 35 on the header) and `RXD = P3.7` (pin 34 on the header) at 115,200 Baud.
 
 The easiest way is to connect an USB TTL adapter (such as the cheap `cp210x`
 or `ch341` based adapters) as follows:
 
 ```
-TTL adapter    Olimex MSP430-H2618
+TTL adapter    Olimex MSP430-H1611
```

Comment:
But the board name is `olimex-msp430-h2618`

Reply:
Good catch. Fixed